hacks_mbof/urls.py

```diff
@@ -43,7 +45,20 @@
     url(r'^bower_components/(?P<path>.*)$', serve, {
         'document_root': 'mbofui/bower_components',
     }),
+]
+
+if 'djangosaml2' in settings.INSTALLED_APPS:
+    urlpatterns += (
+        url(r'^accounts/', include('djangosaml2.urls')),
+        url(r'^test/', 'djangosaml2.views.echo_attributes'),
+    )
```

Comment:
So, I wonder...  Why are these `accounts` and `test` URLs being added as a tuple (inside parentheses) rather than a list (inside square brackets)?  Try using a list for these (and those below), too.